mitosheet/mitosheet/telemetry/telemetry_utils.py

```diff
@@ -391,4 +394,14 @@ def log(log_event: str, params: Optional[Dict[str, Any]]=None, steps_manager: Op
         print(
             log_event, 
             final_params
+        )
+
+    analytics_url = steps_manager.mito_config.get_analytics_url() if steps_manager is not None else None
+    if analytics_url is not None:
+        requests.post(
+            analytics_url,
+            json={
+                'user_id': get_user_field(UJ_STATIC_USER_ID),
+                'log_event': log_event
```

Comment:
Do we want to add the final_params, here?

Reply:
I think no, not for v1.